internal/suites/scenario_one_factor_test.go

```diff
@@ -136,5 +136,5 @@ func TestRunOneFactor(t *testing.T) {
 		t.Skip("skipping suite test in short mode")
 	}
 
-	suite.Run(t, New1FAScenario())
+	suite.Run(t, NewOneFactorScenario())
```

Comment:
This was named `1FA` instead of `OneFactor` to ensure ordering, if it was a nit I would likely just revert the majority of this change (happy to change Suite -> Scenario).